test/User-test.js

```diff
@@ -1,31 +1,31 @@
 import { expect } from 'chai';
-import User from '../src/classes/User.js';
-import Recipe from '../src/classes/Recipe.js';
-import { recipeData } from '../src/data/recipes.js';
-import { users } from '../src/data/users.js';
+import User from '../src/classes/User';
+import Recipe from '../src/classes/Recipe';
+import { usersData } from '../src/data/users-sample';
+import { recipeData } from '../src/data/recipes-sample';
+import { ingredientsData } from '../src/data/ingredients-sample';
 
 describe('User', function() {
-  let user;
-  let recipeOne;
-  let recipeTwo;
-  let recipeThree;
-  let tagOne;
-  let tagTwo;
+  let user, recipeOne, recipeTwo, recipeThree, tagOne, tagTwo, tagThree, ingredientsOne, ingredientsTwo, ingredientsThree, nameOne;
 
-  beforeEach(() => {
-    user = new User(users[0]);
-    recipeOne = new Recipe(recipeData[7]);
-    recipeTwo = new Recipe(recipeData[3]);
+  beforeEach(function() {
+    user = new User(usersData[0]);
+    recipeOne = new Recipe(recipeData[2]);
+    recipeTwo = new Recipe(recipeData[1]);
     recipeThree = new Recipe(recipeData[0]);
     tagOne = 'appetizer'
     tagTwo = 'snack'
+    tagThree = 'side dish'
+    nameOne = 'cookies'
```

Comment:
Just for clarity on how `beforeEach` works, it allows you to "reset" things before each test. This is useful for things that hold onto "state" like classes, which is why `beforeEach` is awesome for stuff like `user, recipeOne, recipeTwo, etc.`. However, `tagOne, tageTwo, nameOne, etc.` never change. So, you couldn't technically make these `const` outside of the `beforeEach` or even outside of the `describe` block.

No real need to change, but food for thought.